cmd/kube-apiserver/app/aggregator.go

```diff
@@ -182,7 +183,7 @@ func apiServicesToRegister(delegateAPIServer genericapiserver.DelegationTarget,
 		// but for now its a special case
 		// apps has to come last for compatibility with 1.5 kubectl clients
 		if apiService.Spec.Group == "apps" {
```

Comment:
actually, I think moving extensions earlier would be a better approach... the only order we've cared about so far is with resources that exist both in extensions and in their final group